haskell/private/actions/link.bzl

```diff
@@ -227,11 +230,12 @@ def _add_external_libraries(args, libs):
                 "-L{0}".format(paths.dirname(lib.path)),
             ])
 
-def _infer_rpaths(target, solibs):
+def _infer_rpaths(hs, target, solibs):
     """Return set of RPATH values to be added to target so it can find all
     solibs.
 
     Args:
+      hs: Haskell context.
```

Comment:
Adding `hs` is slightly overkill in this case, you could pass `is_darwin` directly.

Reply:
Done.